### ISSUE TITLE
User sign-up creates a admin user within it's own namespace since V2.3.0

### ISSUE THREAD

**Issue by prokofyevDmitry** [lowcoder-org/lowcoder#656]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When creating a new user (NOT the first one since setup of the server)  through the `sing-up` menu (because our server has ENABLE_USER_SIGN_UP set to true) the created user has admin rights inside his own workspace that is as well created on user sign up. 
That means that a random person can sign-up and start creating views in it's own workspace (which can mean he can launch big requests that would load the server). 
It seems that this behaviour appeared after the upgrade of our service to v2.3  from v2.2.1.

### Expected Behavior

The user is created as Member in a certain workspace from which he is then authorized the access/assigned new Role/invited in another workspace by an administrator. 
The sign-up should only create one admin user: the first one of the whole setup. 

### Steps to reproduce

![new-user-creation](https://github.com/lowcoder-org/lowcoder/assets/8905771/d3c89401-839b-4a03-9b2a-c115e1a8b72c)

### Environment

Bug found in the docker version 2.3.0: https://hub.docker.com/layers/lowcoderorg/lowcoder-ce/latest/images/sha256-f311b16ca45c59423ae92c12943fa2150c7a36e2558ec8bf4b14ea729f3e0db9?context=explore

Instance environment variables: 
ENV REDIS_ENABLED="true" \
    MONGODB_ENABLED="true" \
    API_SERVICE_ENABLED="true" \
    NODE_SERVICE_ENABLED="true" \
    FRONTEND_ENABLED="true" \
    PUID="1000" \
    PGID="1000" \
    DEFAULT_ORGS_PER_USER=100 \
    DEFAULT_ORG_MEMBER_COUNT=1000 \
    DEFAULT_ORG_GROUP_COUNT=100 \
    DEFAULT_ORG_APP_COUNT=1000 \
    DEFAULT_DEVELOPER_COUNT=50 \
    MONGODB_URL="mongodb://localhost:27017/lowcoder?authSource=admin" \
    REDIS_URL="redis://localhost:6379" \
    ENABLE_USER_SIGN_UP="true" \
    ENCRYPTION_PASSWORD=XXXXXXXXX
    ENCRYPTION_SALT=XXXXXXXXX
    CORS_ALLOWED_DOMAINS="*" \
    LOWCODER_API_KEY_SECRET=XXXXXXXXX \
    LOWCODER_API_SERVICE_URL="http://localhost:8080" \
    LOWCODER_NODE_SERVICE_URL="http://localhost:6060" \
    LOWCODER_MAX_REQUEST_SIZE="20m" \
    LOWCODER_MAX_QUERY_TIMEOUT=120

**Comment by prokofyevDmitry** [comment]
Ok I found that there was some extensive modifications on the env var side in this update: 
![image](https://github.com/lowcoder-org/lowcoder/assets/8905771/e451bb4a-07f0-45d3-b5eb-7a207f7d0e68)

Still think that this should be indicated in large chars in the release notes as people ( for example us) would enable this feature without knowing it by simply updating the docker image without touching the env vars ( if `LOWCODER_CREATE_WORKSPACE_ON_SIGNUP` is not defined, it defaults to true)

**Comment by FalkWolsky** [comment]
Thank you for creating the issue. As you found, we invested time in cleaning up the ENV variables.
In the release notes, we did describe that we have breaking changes - and on Discord we stated about the ENV Variables too. What is left to be done and here you are right - we need urgently to update the documentation to show how the system behaves with the different variations of the ENV Vars to control the behavior.

Cause what you describe as a bug - was nothing else than the wishful standard behavior for most of the users. So, it is more a misunderstanding. We let the issue open, till we have done a proper docu.
